frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.test.tsx

```diff
@@ -4,20 +4,28 @@ import {
   screen,
   waitForElementToBeRemoved,
 } from "@testing-library/react";
-import { rest } from "msw";
-import { server } from "../../../mocks/msw/server";
 import { PlayerProfileDto } from "../../../restapi/players-profiles";
 import userEvent from "@testing-library/user-event";
-import {BrowserRouter as Router} from "react-router-dom";
-
+import { BrowserRouter as Router } from "react-router-dom";
+import {
+  getPlayersProfilesIsLoading,
+  getPlayersProfilesWillReturn,
+} from "../../../restapi/players-profiles/PlayersProfilesRestApiMockEndpoints";
+import { getRegisteredPlayersIdsWillReturn } from "../../../restapi/tournament-registrations/TournamentRegistrationsRestApiMockEndpoints";
 
 describe("Tournament Registrations", () => {
+  const tournamentId = "tournamentId";
   it(`should show title "Zapisy na turniej"`, () => {
     //Given
     getPlayersProfilesIsLoading();
+    getRegisteredPlayersIdsWillReturn(tournamentId, "open", []);
 
     //When
-    render(<Router><TournamentRegistrations/></Router>);
+    render(
+      <Router>
```

Comment:
Wszędzie Router jest niepotrzebny. Zdaje mi się, że nie wiesz co to robi :D To co to robi?

Reply:
Mati, ale to są Twoje testy ^_^

Reply:
https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/blob/develop/frontend/src/components/pages/TournamentRegistrations/TournamentRegistrations.test.tsx

Reply:
ja tylko posłusznie i bezmyślnie skopiowałem dalej :-P do moich testów

Reply:
no już sobie poczytałem, że raczej niepotrzebne:
https://reactrouter.com/web/api/BrowserRouter
https://reactrouter.com/core/api/Router

Reply:
1. To nie są moje testy. Bo to jest nasz wspólny projekt :) 
2. Nie chcę bezmyślnych wyznawców :D To, że ja coś zrobiłem nie znaczy, że jest dobrze / nie można zrobić lepiej. 
3. Git blame - nie ja to dopisałem (a nawet masz autora przy commicie) :) I wtedy jeszcze było potrzebne, po ostatnich zmianach nie jest.  No bo wysunąłeś obsługę routera do innego komponentu. No i ułatwiłeś też testy :)

Reply:
1. a dobra dobra to zwracam honor :-P ... no już to było tu, to tak zalożyłem, a nie spojrzałem :-P
2.  ale zazwyczaj tak jest, więc cóż poradzić
3. a no dobra, bo jest to <Route> przeniesione do pliku TourDeFoos, racja :-)